.github/workflows/verify-bundled-files.yml

```diff
@@ -0,0 +1,43 @@
+name: Verify Bundled Files
+
+on:
+  push:
+  pull_request:
+  schedule:
+   - cron: "0 1 * * *"
+  workflow_dispatch: ~
+
+permissions:
+  contents: read
+
+jobs:
+  VERIFY_BUNDLED_FILES:
+    name: Verify Bundled Files
+    runs-on: ubuntu-22.04
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v5
+
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          base: master
+          filters: |
+            opcache_jit_ir:
+              - 'ext/opcache/jit/ir/**'
+
+      - name: Opcache JIT IR - Download
+        if: ${{ !cancelled() && (steps.changes.outputs.opcache_jit_ir == 'true' || (github.event_name != 'push' && github.event_name != 'pull_request')) }}
+        uses: actions/checkout@v5
+        with:
+          repository: dstogov/ir
+          ref: 5a81104e650ebd7ac24eb63d4dff67db723a5278
+          path: ext/opcache/jit/ir
+
+      - name: Opcache JIT IR - Verify files
+        if: ${{ !cancelled() && (steps.changes.outputs.opcache_jit_ir == 'true' || (github.event_name != 'push' && github.event_name != 'pull_request')) }}
+        run: |
+          cd ext/opcache/jit/ir
+          rm -r .git .github/workflows bench examples tests tools README.md TODO ir.g ir_cpuinfo.c ir_emit_c.c ir_emit_llvm.c ir_load.c ir_load_llvm.c ir_main.c ir_mem2ssa.c
+          git restore README
+          git add . -N && git diff --cached -a --exit-code . && git diff -a --exit-code .
```

Comment:
This comparison doesn't look robust. It's going to be broken every time when a new IR file is introduced.
Instead of using GIT tricks, the comparison could be implemented using ``rsync`` or just ``md5sum``.

Reply:
It won't be broken when a new IR file is introduced. It will be broken when a new "not copied" file is introduced.

Currently there are several files not copied and these files are ignored manually.